### PR TITLE
Code issues : Avoid code duplicate

### PR DESCRIPTION
**Pull request by PululuK**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This change fixes minor code issues I found while working on another bug.
| Type?         | refacto
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A
| How to test?  | Make sure that `BO > International > Localization > Languages` work !!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/19780)

### CODE REVIEW

File: src/Adapter/Language/CommandHandler/BulkToggleLanguagesStatusHandler.php
Comment (Progi1984) [inline src/Adapter/Language/CommandHandler/BulkToggleLanguagesStatusHandler.php]: I don't agree to remove the negative

File: src/Adapter/Language/CommandHandler/ToggleLanguageStatusHandler.php
Comment (Progi1984) [inline src/Adapter/Language/CommandHandler/ToggleLanguageStatusHandler.php]: I don't agree to remove the negative

File: src/Adapter/Language/CommandHandler/AbstractLanguageHandler.php
Comment (PululuK) [inline src/Adapter/Language/CommandHandler/AbstractLanguageHandler.php]: Renamed to avoid a conflict with https://github.com/PrestaShop/PrestaShop/blob/develop/src/Adapter/Language/CommandHandler/ToggleLanguageStatusHandler.php#L63
 In any case I prepare another PR to improve the Toogles classes too.

Comment (Progi1984) [reply src/Adapter/Language/CommandHandler/AbstractLanguageHandler.php]: Perfect.

**Review by Progi1984 — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by matthieu-rolland — COMMENTED** [review]
Thanx for this @PululuK :) 

You need to run PHP cs fixer on these files:

`src/Core/Domain/Language/Command/BulkToggleLanguagesStatusCommand.php`
`src/Core/Domain/Language/Command/ToggleLanguageStatusCommand.php`
`src/Adapter/Language/CommandHandler/BulkToggleLanguagesStatusHandler.php`
`src/Adapter/Language/CommandHandler/ToggleLanguageStatusHandler.php`
`src/Adapter/Language/CommandHandler/AbstractLanguageHandler.php`

(If for some reason it's not convenient for you to do it ping me and I'll do it on your PR ;)  )

**Comment by PululuK** [comment]
> Thanx for this @PululuK :)
> 
> You need to run PHP cs fixer on these files:
> 
> `src/Core/Domain/Language/Command/BulkToggleLanguagesStatusCommand.php`
> `src/Core/Domain/Language/Command/ToggleLanguageStatusCommand.php`
> `src/Adapter/Language/CommandHandler/BulkToggleLanguagesStatusHandler.php`
> `src/Adapter/Language/CommandHandler/ToggleLanguageStatusHandler.php`
> `src/Adapter/Language/CommandHandler/AbstractLanguageHandler.php`
> 
> (If for some reason it's not convenient for you to do it ping me and I'll do it on your PR ;) )

Can you do it please ? Big  thanks @matthieu-rolland !

**Review by matthieu-rolland — DISMISSED** [review]
LGTM,  I'll put the "waiting for QA label" once the CI is OK 👍  (and once we have a second approve)

**Review by matks — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

File: src/Adapter/Language/CommandHandler/AbstractLanguageHandler.php
Comment (jolelievre) [inline src/Adapter/Language/CommandHandler/AbstractLanguageHandler.php]: If you want to use this in other classes it needs to be `protected` no?

Comment (PululuK) [reply src/Adapter/Language/CommandHandler/AbstractLanguageHandler.php]: Exactly ! But it's very strange how Travis didn't catch it ? Thank you @jolelievre

**Comment by SD1982** [comment]
LGTM the page  "BO > International > Localization > Languages" seems to work as before ! Thanks @PululuK

**Review by matks — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @PululuK & @SD1982

**Pull request merged**
